music/admin.py

```diff
@@ -17,4 +27,18 @@ def add_music(self, request, queryset):
             music.room.add_music(**MusicSerializer(music).data)
         return
 
+    def duplicate_music(self,request,queryset):
+        targetRoom = Room.objects.get(name=request.POST['nrroom'])
+        if targetRoom:
+            for music in queryset:
+                if targetRoom.name != music.room.name:
+                    new_entry = music
```

Comment:
This does not duplicate the music. new_entry is just a reference to the same music object. You have to a new create one with same values